src/server.test.js

```diff
@@ -0,0 +1,22 @@
+import hapi from "@hapi/hapi";
+import { describe, expect, it, vi } from "vitest";
+import { createServer } from "./server.js";
+
+vi.mock("@hapi/hapi");
+
+describe("server", () => {
+  it("registers server", async () => {
+    const on = vi.fn();
+    const register = vi.fn();
+    vi.spyOn(hapi, "server").mockImplementation(() => ({
+      events: {
+        on,
+      },
+      register,
+    }));
+
+    await createServer();
+    expect(on).toHaveBeenCalledTimes(2);
+    expect(register).toHaveBeenCalledTimes(2);
+  });
```

Comment:
Instead of faking coverage, can we initialise the server and test the same as the CW frontend and backend do?